pkgs/development/python-modules/dnf4/default.nix

```diff
@@ -36,7 +40,8 @@ buildPythonPackage rec {
     substituteInPlace etc/tmpfiles.d/CMakeLists.txt \
       --replace "DESTINATION /usr/lib/tmpfiles.d" "DESTINATION $out/usr/lib/tmpfiles.d"
     substituteInPlace dnf/const.py.in \
-      --replace "/etc" "$out/etc"
+      --replace "/etc" "$out/etc" \
```

Comment:
I think `python310Packages.dnf4/etc/bash_completion.d/dnf` should be moved to `share/bash-completion/completions/dnf`

Reply:
I've added a step to move the completion of dnf4 in postInstall. dnf-plugins-core doesn't seem to have a completion script, so not sure what other packages you are referring to.